example/src/screens/Examples/AwareScrollView/index.tsx

```diff
@@ -15,6 +16,8 @@ export default function AwareScrollView({ navigation }: Props) {
   const [disableScrollOnKeyboardHide, setDisableScrollOnKeyboardHide] =
     useState(false);
 
+  const isFocused = useIsFocused();
```

Comment:
Ideally would be good to add a button somewhere in header to be able to toggle this property and cover it by e2e tests 🙏